src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor().name();
+        int number = new Random().nextInt(MAX_BALL_NUMBER);
```

Comment:
Think about which variables should be local in the method and which should be class-level

see https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery